src/nl/surf/eduhub/validator/service/main.clj

```diff
@@ -99,5 +102,6 @@
     (start-server (-> app-routes
                       (wrap-validator config)
                       (auth/wrap-authentication introspection-endpoint introspection-auth)
+                      (auth/wrap-allowed-clients-checker allowed-client-ids)
```

Comment:
Authentication moet onder allowed-clients-checker -- we willen de request testen